.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 - [ ] closes #xxxx (Replace xxxx with the Github issue number)
+- [ ] xref #xxxx (Replace xxxx with the Github issue number)
```

Comment:
Why was this added? I am concern that it may be confusing with there also being `closes #xxxx above`

IMO we want to encourage the users to use `closes #xxxx` if applicable so it automatically closes the linked issue

Reply:
Yeah agreed. There are already many checkboxes you have to tick, would prefer to not add this